### PR TITLE
Add suport for "standard" NO_COLOR env variable

### PR DESCRIPTION
**Pull request by leovilok**
See https://no-color.org

### CODE REVIEW

File: src/nnn.c [hunk 1]
```diff
@@ -6867,6 +6867,9 @@ int main(int argc, char *argv[])
 		}
 	}
 
+	if (getenv("NO_COLOR"))
```
Comment (jarun) [inline src/nnn.c:6870]: Not here. Add in `initcurses()`:

```diff
diff --git a/src/nnn.c b/src/nnn.c
index 20f42b6..1609343 100644
--- a/src/nnn.c
+++ b/src/nnn.c
@@ -1524,7 +1524,7 @@ static bool initcurses(void *oldmask)
 #endif
        curs_set(FALSE); /* Hide cursor */
 
-       if (!(g_states & STATE_NOCOLOR)) {
+       if (!((g_states & STATE_NOCOLOR) || getenv("NO_COLOR"))) {
                start_color();
                use_default_colors();
 
```

**Comment by jarun** [comment]
There's a list of utils in the link. See if they would add `nnn` to the list. This is useful!

File: src/nnn.c (hunk 1 above)
Comment (leovilok) [reply src/nnn.c:6870]: done

**Comment by leovilok** [comment]
> There's a list of utils in the link. See if they would add `nnn` to the list. This is useful!

I'm preparing a PR on their repo, if you want.

**Comment by jarun** [comment]
> I'm preparing a PR on their repo, if you want.

Sure thing!

File: src/nnn.c (hunk 1 above)
Comment (leovilok) [reply src/nnn.c]: done, this time ...

**Comment by jarun** [comment]
@leovilok please wait for someone else in the team to review the final changes and do the merge.

**Comment by leovilok** [comment]
@jarun sorry, I needed a commit ID to make the no_color PR: https://github.com/jcs/no_color/pull/76

**Comment by jarun** [comment]
No problem! Everyone does this the first time. ;)

**Comment by jarun** [comment]
Thank you!

**Comment by leovilok** [comment]
@jarun `nnn` is now listed on https://no-color.org/

**Comment by jarun** [comment]
Awesome! Thank you!

**Comment by lawnowner** [comment]
@leovilok @jarun The informal NO_COLOR standard allows the users to have colored output in their applications **only if** they want to, which is a good idea, but it's not a per-application setting, and thus the applications that can output colored text and also support the standard are supposed to provide an option to explicitly enable color. Also stated in their website: 

> User-level configuration files and per-instance command-line arguments should override $NO_COLOR.

Right now, NO_COLOR overrides NNN_COLORS.

Also, congratulations for the anniversary!

**Comment by jarun** [comment]
I think we should honor `NNN_COLORS`, if it's not defined then honor `NO_COLOR`, if none of the 2 defined show default colors.

@leovilok can you please take care of this? No hurry but please keep the change minimal.

**Comment by leovilok** [comment]
@jarun @lawnowner see https://github.com/jarun/nnn/pull/515

**Pull request merged**